src/squlearn/encoding_circuit/circuit_library/chebyshev_pqc.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
for what is that needed?

Reply:
for the return type hint in `set_params`